src/Volume.ts

```diff
@@ -270,7 +283,7 @@ export default class Volume {
     if (this.channels.every((element) => element.loaded)) {
       this.loaded = true;
     }
-    batch.forEach((channelIndex) => this.channelLoadCallback?.(this.loadSpec.url, this, channelIndex));
+    batch.forEach((channelIndex) => this.channelLoadCallback?.(this, channelIndex));
```

Comment:
Just asking for my own clarification, does the `function?.(args)` mean you're only calling the function if it exists?

Reply:
Yep!